flexmeasures/api/v3_0/assets.py

```diff
@@ -1107,6 +1107,67 @@ def update_default_asset_view(self, **kwargs):
             "message": "Default asset view updated successfully.",
         }, 200
 
+    @route("/default_legend_position", methods=["POST"])
+    @as_json
+    @use_kwargs(
+        {"default_legend_position": fields.Boolean(required=False)}, location="json"
+    )
+    def update_default_legend_position(self, **kwargs):
+        """
+        .. :quickref: Assets; Update the default legend position for graphs for the current user
+        ---
+        post:
+          summary: Update the default legend position for graphs for the current user
+          description: |
+            This endpoint sets the default legend position for graphs either at the right or at the bottom of the chart. The bottom (a combined legend) is the default.
+          security:
+            - ApiKeyAuth: []
+          requestBody:
+            required: true
+            content:
+              application/json:
+                schema:
+                  type: object
+                  properties:
+                    default_legend_position:
+                      type: boolean
+                      description: Whether to use the default legend position (bottom) or not (right). Right is only used if there are several sensors on the graphs page, in any case.
```

Comment:
Do we really need to repeat the description here? I believe it can go - the endpoint description says enough, and the endpoint is just this field.